sdk/storage/Azure.Storage.DataMovement/tests/BlobStorageResourceContainerTests.cs

```diff
@@ -254,7 +254,7 @@ public async Task GetChildStorageResourceAsync()
             StorageResourceContainer containerResource =
                 new BlobStorageResourceContainer(test.Container, new() { BlobDirectoryPrefix = prefix });
 
-            StorageResourceItem resource = containerResource.GetStorageResourceReference("bar");
+            StorageResourceItem resource = containerResource.GetStorageResourceReference("bar", default);
```

Comment:
nit: For these test files also, should we pass something other than `default`?

Reply:
Below I added tests where we don't pass just default.